src/App.jsx

```diff
@@ -1,35 +1,296 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import styled from "styled-components";
+import GlobalStyle from "./GlobalStyleCSS";
+
+import * as brandLogos from "./assets/brandLogos/brandLogos";
+
+import HomePage from "./pages/Home";
+import NewsPage from "./pages/News";
+import EventsPage from "./pages/Events";
+import ContactPage from "./pages/Contact";
+import NotFoundPage from "./pages/NotFound";
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #111928;
+
+  & > * {
+    padding-top: 96px;
+    padding-bottom: 96px;
+  }
+
+  & > :first-child {
+    padding-top: 24px;
+    padding-bottom: 0px;
+  }
+
+  & > :last-child {
+    padding-top: 96px;
+    padding-bottom: 64px;
+  }
+`;
+
+export const SpecialCointaner = styled(Container)`
+  padding-top: 0px;
+
+  & > :last-child {
+    padding-top: 0px;
+    padding-bottom: 0px;
+  }
+`;
+
+const projectData = {
```

Comment:
ciekawe podejście do przechowywania danych, mi się podoba, choć w przyszłości pewnie będzie to nieco bardziej rozbite na różne źródła i zapytania